Added NullCon 2020 (#176)
nikhita/tech-conferences-india: year-pages/2020.md

| [Machine Learning Developers Summit](https://www.mlds.analyticsindiasummit.com/) | 30-31 Jan | Indian School of Business, Hyderabad | India’s leading Machine Learning innovators and practitioners to share their ideas and experience about machine learning tools, advanced development & developer products | No |
| [Machine Learning Developers Summit](https://www.mlds.analyticsindiasummit.com/) | 22-23 Jan | NIMHANS Convention Center, Bengaluru | India’s leading Machine Learning innovators and practitioners to share their ideas and experience about machine learning tools, advanced development & developer products | No |
| [India OS](https://indiaos.in/) | 18 Jan | Zerodha, Bengaluru, Karnataka | See what's shaping the future of software, join a community connected by code and turn ideas into innovations. The goal is to provide software developers and policy makers, communities and enthusiasts a platform to meet and interact. | Free to attend |

> | [NullCon](https://nullcon.net/website/goa-2020/cfp.php) | 06 March | Taj Hotel and Convention Center, Dona Paula | Nullcon is an annual security conference held in Goa, India. The focus of the conference is to showcase the next generation of offensive and defensive security technology. We happily open doors to researchers and hackers around the world and the universe, working on the next big thing in security and request everyone to submit their new research. | No |

Reviewer: ```suggestion
| [NullCon](https://nullcon.net/website/goa-2020/cfp.php) | 06 Mar | Taj Hotel and Convention Center, Dona Paula | Nullcon is an annual security conference held in Goa, India. The focus of the conference is to showcase the next generation of offensive and defensive security technology. | No |
```

Reviewer: Oh, also please move this conference near the top of the README to maintain the order. Thanks!

Author: Sure.
